### PR TITLE
Run Nessie in parallel on each cloud

### PR DESCRIPTION
**Pull request by arielshaqed**
Generate code _once_, then run Nessie on each cloud separately.  Should be
faster.

### CODE REVIEW

**Comment by arielshaqed** [comment]
This reduces time to run Nessie by 3-5 minutes, I see 12m14s (single run) instead of 14m32s (once) and generally 16-20 minutes on the previous version.

**Comment by arielshaqed** [comment]
Removing "Run lakeFS S3 to use with local API key" would reduce runtime by another 2m.  Or I could parallelize that _too_, making those 2m run in parallel, and reducing runtime by a bit less while keeping the same coverage.

@itaiad200 WDYT?

File: .github/workflows/nessie.yaml [hunk 1]
```diff
@@ -287,6 +306,32 @@ jobs:
...
+        run: tar -xf /tmp/generated.tar.gz
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
```
Comment (itaiad200) [inline .github/workflows/nessie.yaml:334]: Why do we need aws creds for gs?

File: .github/workflows/nessie.yaml [hunk 2]
```diff
@@ -304,6 +349,33 @@ jobs:
...
```
Comment (itaiad200) [inline .github/workflows/nessie.yaml:377]: Why do we need aws creds for Azure?

**Review by itaiad200 — APPROVED** [review]

File: .github/workflows/nessie.yaml (hunk 1 above)
Comment (arielshaqed) [reply .github/workflows/nessie.yaml:334]: Fetch from ECR.  AFAIK that repo is not open for public read.  E.g. if I don't do the `aws ecr get-login ...; docker login` dance then Docker cannot fetch images:
```sh
ariels@redqueen:~/dev/lakeFS$ docker pull 977611293394.dkr.ecr.us-east-1.amazonaws.com/lakefs:latest
Error response from daemon: Head https://977611293394.dkr.ecr.us-east-1.amazonaws.com/v2/lakefs/manifests/latest: no basic auth credentials
```

We might ditch ECR (for this, at least) and just use the GitHub Docker registry.  But that would be in a separate PR.

File: .github/workflows/nessie.yaml (hunk 2 above)
Comment (arielshaqed) [reply .github/workflows/nessie.yaml:377]: Same same.  >:-)

**Review by arielshaqed — COMMENTED** [review]
Thanks!

**Pull request merged**